discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -131,9 +131,9 @@ def _download(self, response, file_path, verify_file_size=False):
                     f'Actual size: {actual_file_size}. Expected size: {content_length}'
                     )
 
-    def generate_image(self, cluster_id, ssh_key, static_ips=None):
+    def generate_image(self, cluster_id, ssh_key, image_type, static_ips=None):
```

Comment:
Any certain reason why `generate_and_download_image` has `image_type` with a default value and `generate_image` doesn't? I think we could have the same signature (either both have a default or neither of them)

Reply:
Just cause generate_image is called only locally (in class), but sure can change to keep consistency.